bin/do_deltas.py

```diff
@@ -115,6 +118,27 @@ def cont_fit(data):
     data,ndata = io.read_data(args.in_dir,args.drq,args.mode,\
             zmin=args.zqso_min,zmax=args.zqso_max,nspec=args.nspec,log=log,keep_bal=args.keep_bal)
     
+
+    ### Veto sky lines
+    if (args.veto_line_file is not None):
+        try:
+            veto_line = []
+            with open(args.veto_line_file, 'r') as f:
+                loop = True
+                for l in f:
+                    l = l.split()
+                    veto_line += [ [float(l[1]),float(l[2])] ]
```

Comment:
maybe usr_mask is a better name for veto_line. Also consider skipping lines starting with '#' so people can add comments